### PR TITLE
ci: add edge-bonsai tag to image release

### PR DESCRIPTION
**Pull request by worldofgeese**
**What this PR does / why we need it**:

For testing edge-bonsai it would be good to build and release container images for use where CI depends on a container image e.g. GitLab CI. Users sourcing the Garden binary as part of a multi-stage Dockerfile strategy should also be pleased with the option to have an edge release available.

**Special notes for your reviewer**:

Literally my first commit to Garden: be gentle 🌷

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1074,14 +1079,21 @@ workflows:
...
 
+      ### MAIN AND 0.13 ###
+      - release-service-docker:
+          context: docker
+          requires: [build-dist-edge]
+          filters:
+            branches:
+              only:
+                - main
+                - /edge-bonsai.*/
```
Comment (Walther) [inline .circleci/config.yml:1094]: should this be `0.13`?

Comment (worldofgeese) [reply .circleci/config.yml:1094]: Yes 😄

**Review by Walther — DISMISSED** [review]
Thank you for your first contribution to garden core! 🌷 
Looking good, let's hope the CI pipeline likes it as well <3

**Review by Walther — APPROVED** [review]

**Review by Walther — APPROVED** [review]

**Comment by vvagaytsev** [comment]
This has already been merged into `0.13`, and the changes from this PR are not necessary for the `main` branch. Closing this.

**Pull request closed without merging**